src/main/java/com/blamejared/crafttweaker/api/zencode/impl/registry/BracketResolverRegistry.java

```diff
@@ -99,10 +100,10 @@ private void addBracketResolver(Method method) {
             isValid = false;
         }
     
-        if(!CommandStringDisplayable.class.isAssignableFrom(method.getReturnType())){
-            CraftTweakerAPI.logError("Method \"%s\" is marked as a BracketResolver, so it should return something that implements %s.", method.toString(), CommandStringDisplayable.class.getSimpleName());
-            isValid = false;
-        }
+        //if(!CommandStringDisplayable.class.isAssignableFrom(method.getReturnType())){
+        //    CraftTweakerAPI.logError("Method \"%s\" is marked as a BracketResolver, so it should return something that implements %s.", method.toString(), CommandStringDisplayable.class.getSimpleName());
+        //    isValid = false;
+        //}
```

Comment:
Why is this commented out?

Reply:
Since we now allow BEPs to return native types, that check is invalid.
I was wondering whether we should leave it out completely, or replace it with something that checks if the return type has a getCommandString method?

Reply:
Ahh okay, I was just wondering why it was commented out, don't think we need to complicate it